app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.scss

```diff
@@ -60,12 +60,8 @@
 
     &.not-connection {
       background-color: #ffffff;
-      border-width: 10px;
-      padding: 5px 0px;
-      border-radius: 100px;
-      height: 33px;
-      margin: 11px;
-      width: 32px;
+      height: 100%;
+      width: 100%;
```

Comment:
I think the height and width here are unnecessary. This icon should get width/height 100% from https://github.com/syndesisio/syndesis/blob/cd59a1b99ebcce04f66290f9d559084adfcf947f/app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.scss#L53

Reply:
- property and values width/height has been removed. selector inherits width: 100% from parent (`&`).